src/app/(landing)/_components/FAQs/index.tsx

```diff
@@ -1,37 +1,60 @@
-import React from 'react'
-import cn from 'clsx'
+import React, { useState, useEffect } from 'react';
+import cn from 'clsx';
 
-import faqsData from '../../_data/faqs-data'
+import faqsData from '../../_data/faqs-data';
+import { CLASS_NAME } from './const';
+import { type FAQsProps } from './types';
 
-import { CLASS_NAME } from './const'
-import { type FAQsProps } from './types'
+const FAQs: React.FC<FAQsProps> = (props: FAQsProps): JSX.Element => {
+  const { className } = props;
+  const [openIndex, setOpenIndex] = useState<number | null>(null);
 
-import './style.css'
+  useEffect(() => {
+    const detailsElements = document.querySelectorAll('details');
 
-const FAQs: React.FC<FAQsProps> = (props: FAQsProps): JSX.Element => {
-  const { className } = props
+    detailsElements.forEach((detail, index) => {
+      detail.addEventListener('toggle', () => {
+        setOpenIndex(detail.open ? index : null);
+      });
+    });
+
+    return () => {
+      detailsElements.forEach((detail) => {
+        detail.removeEventListener('toggle', () => {});
+      });
+    };
+  }, []);
 
   return (
     <div className={cn(CLASS_NAME, className)}>
-      <div className="blue-neon-mist faq-mist-1"></div>
-      <h2 className="faqs-heading">
-        <span>FAQ</span>s
+      <div className="absolute h-[300px] w-[300px] left-1/2 top-[300px] transform -translate-x-1/2 -z-10 bg-transparent"></div>
+      <h2 className="text-center">
+        <span className="text-primary-blue uppercase">FAQ</span>s
       </h2>
-      <p className="faqs-des px-4">
+      <p className="text-center px-4">
         We are here to help you with any question you have
       </p>
-      <div className="questions-list">
+      <div className="sm:px-[150px] sm:mt-[25px] px-[20px]">
         {faqsData.map((faq, index) => (
-          <details className="question" key={index} open={false}>
-            <summary className="question-head">{faq.question}</summary>
-            <div className="question-des">
+          <details
+            key={index}
+            className={`bg-background-light p-[10px] rounded-md shadow-custom-light mb-[15px] transition-all duration-300 ease-in-out
```

Comment:
please give meaningful variables, instead of "custom-light" or "custom-dark", try use something like "faq-shadow-light", just an example, do not add unnecessary variables into the global scope,